contracts/SnapshotToken/SnapshotToken.sol

```diff
@@ -90,9 +95,30 @@ contract SnapshotToken is
         public
         returns (bool success)
     {
+        // NOTE: We do not call the ERC223 callback
+        // here for compatibility reasons. Please use
+        // tranfser(_to, _amount, bytes()) instead.
         return transfer(msg.sender, _to, _amount);
     }
 
+    function transfer(address to, uint amount, bytes data)
+        public
+        returns (bool success)
+    {
+        success = transfer(msg.sender, to, amount);
+        if (!success) {
+            return success;
+        }
+
+        // Notify the receiving contract.
+        if (isContract(to)) {
+            IERC223Callback(to).tokenFallback(msg.sender, amount, data);
+        }
+
+        Transfer(msg.sender, to, amount, data);
```

Comment:
I do not like the idea to have two Transfer events. `data` is not affecting token contract in any way so why log it? If it has any impact in fallback, the receiving contract can log accordingly. Standard has just one event but it is a breaking change and I do not like it even more.

Reply:
I also don't like it for exactly the reasons you mention, but ERC223 requires it to have data in there. I don't think it will be a huge issue if we also break away from ERC223 on this one. Shall I remove the extra event?